proto/kvrpcpb.proto

```diff
@@ -1119,6 +1119,18 @@ message GetLockWaitInfoResponse {
     repeated deadlock.WaitForEntry entries = 3;
 }
 
+message GetLockWaitHistoryRequest {
+    Context context = 1;
+    // TODO: There may need some filter options to be used on conditional querying, e.g., finding
+    // the lock waiting status for some specified transaction.
+}
+
+message GetLockWaitHistoryResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    repeated deadlock.WaitForEntry entries = 3;
```

Comment:
The `deadlock.WaitForEntry` type doesn't seem to contain a time. Do we need to record the time when the waiting happens?

Reply:
Seems not easy? I don't know how to get this time... System time on TiKV is not appropriate.

Reply:
Hmmm... I'm still afraid that when a user views the history, the user may want to know approximately when each lock waiting did happen. Without any information about time, it might be a little difficult for a user to figure out which information is helpful for some specific problem. 
Maybe for_update_ts can be a better choice than TiKV's system time? However for_update_ts is not included in the entries either...